play/src/containers/GameSessionContainer.tsx

```diff
@@ -3,24 +3,60 @@ import {
   IGameSession,
   IAWSGameSession,
   GameSessionParser,
+  GameSessionState,
 } from '@righton/networking';
 import MockGameSession from '../mock/MockGameSession.json';
-import GameInProgress from '../pages/GameInProgress'; // eslint-disable-line @typescript-eslint/no-unused-vars
+import GameInProgress from '../pages/GameInProgress';
+import PhaseResults from '../pages/PhaseResults';
 import JoinGame from '../pages/JoinGame';
 import { JoinGameState } from '../lib/PlayModels';
 
 export default function GameSessionContainer() {
-  const [gameSession, setGameSession] = useState(     // eslint-disable-line @typescript-eslint/no-unused-vars
-
+  const [gameSession, setGameSession] = useState( // eslint-disable-line @typescript-eslint/no-unused-vars
     GameSessionParser.gameSessionFromAWSGameSession(
       MockGameSession as IAWSGameSession
     ) as IGameSession
   );
   const [teamAvatar, setTeamAvatar] = useState(0); // eslint-disable-line @typescript-eslint/no-unused-vars
   const [joinGameState, setjoinGameState] = useState<JoinGameState>( // eslint-disable-line @typescript-eslint/no-unused-vars
-    JoinGameState.ENTER_GAME_CODE
+    JoinGameState.SPLASH_SCREEN
   ); 
+  const [gameState, setGameState] = useState<GameSessionState>( // eslint-disable-line @typescript-eslint/no-unused-vars
+    GameSessionState.PHASE_2_RESULTS
+  ); 
+  const [currentQuestionIndex, setCurrentQuestionIndex] = useState<number | null>(0); // eslint-disable-line @typescript-eslint/no-unused-vars
 
-
-  return <JoinGame joinGameState={joinGameState} />;
+  switch (gameState) {
+    case GameSessionState.TEAMS_JOINING:
+      return <JoinGame joinGameState={joinGameState} />;
+    case GameSessionState.CHOOSE_CORRECT_ANSWER:
+    case GameSessionState.CHOOSE_TRICKIEST_ANSWER:
+      return (
+        <GameInProgress
+          {...gameSession}
+          teamAvatar={teamAvatar}
+          teamId="2d609343-de50-4830-b65e-71eb72bb9bef"
+        />
+      );
+    case GameSessionState.PHASE_1_RESULTS:
+    case GameSessionState.PHASE_2_RESULTS:
+      return (
+        <PhaseResults
+          {...gameSession}
+          gameSession={gameSession}
+          currentQuestionIndex={currentQuestionIndex}
+          currentState={gameState}
+          teamAvatar={teamAvatar}
+          teamId="2d609343-de50-4830-b65e-71eb72bb9bef"
+        />
+      );
+    default:
+      return (
+        <GameInProgress
+          {...gameSession}
+          teamAvatar={teamAvatar}
+          teamId="2d609343-de50-4830-b65e-71eb72bb9bef"
+        />
```

Comment:
Not necessary for this PR, but we'd probably wanna check and validate the varaiebls are not `nil` and are available here for each view, like `gameSession` or `teamAvatar` and etc.

Reply:
yup, great idea. I'll add that to the subscription/routing pr. Thanks again for the reviews!